Engine/platform/base/agsplatformdriver.cpp

```diff
@@ -190,18 +190,20 @@ int cd_player_control(int cmdd, int datt) {
 #endif // AGS_HAS_CD_AUDIO
 
 void AGSPlatformDriver::Delay(int millis) {
-  auto delayUntil = AGS_Clock::now() + std::chrono::milliseconds(millis);
+  auto now = AGS_Clock::now();
+  auto delayUntil = now + std::chrono::milliseconds(millis);
 
   for (;;) {
-    if (AGS_Clock::now() >= delayUntil) { break; }
+    if (now >= delayUntil) { break; }
 
-    auto duration = delayUntil - AGS_Clock::now();
-    if (duration > std::chrono::milliseconds(25)) {
-      duration = std::chrono::milliseconds(25);
+    auto duration = delayUntil - now;
+    if (duration > std::chrono::milliseconds(16)) {
+      duration = std::chrono::milliseconds(16);
```

Comment:
Could you add a comment explaining what does 16 ms mean here?

Reply:
Sure! Constants works too.

Reply:
I added the comment. and a constant.